ReactiveArchitecture/Classes/nowplaying/viewcontroller/NowPlayingTableViewController.swift

```diff
@@ -116,25 +116,28 @@ class NowPlayingTableViewController: UITableViewController {
         let movieViewInfo: MovieViewInfo = objectList[indexPath.row]
         
         //Load & Address Cell (no longer returns nil when using storyboard)
-        if (movieViewInfo is MovieViewInfoImpl) {
+        if movieViewInfo is MovieViewInfoImpl {
+            // swiftlint:disable:next force_cast
             let movieCell: MovieCell = tableView.dequeueReusableCell(withIdentifier: "MovieCellIdentifier", for: indexPath) as! MovieCell
```

Comment:
I personally don't mind force casting in this case. A crash here is basically the only sane thing to do as you won't know how the app will behave with a cell of the wrong type

Reply:
:thumbsup: